dashboard/app/controllers/registrations_controller.rb

```diff
@@ -2,6 +2,7 @@ class RegistrationsController < Devise::RegistrationsController
   respond_to :json
 
   def new
+    session[:user_return_to] ||= params[:user_return_to]
```

Comment:
Is it a built-in devise behaviour to use user_return_to or something we've invented?